src/components/Textfield/Textfield.scss

```diff
@@ -0,0 +1,169 @@
+@import "../variables";
```

Comment:
I'm on a case insensitive file system and on my machine this file overwrites the TextField/TextField.scss, so that I only have the incorrect version and the compilation breaks.
You probably have to use `git mv` to resolve this: https://stackoverflow.com/questions/17683458/how-do-i-commit-case-sensitive-only-filename-changes-in-git